2-advanced/dubbo-samples-version/case-versions.conf

```diff
@@ -20,6 +20,6 @@
 # Supported component versions of the test case
 
 # Spring app
-dubbo.version=2.7*, 3.*
+dubbo.version=2.7*, 3.*, !3.1.8*, !3.2.0-beta.6*
```

Comment:
Why ignore this version

Reply:
https://github.com/apache/dubbo/issues/11766